src/main/java/use_cases/monthly_menu/MonthMenuOD.java

```diff
@@ -13,9 +13,9 @@
  * and shown in the Month Menu view.
  */
 public class MonthMenuOD {
```

Comment:
Do I get the arraylist of category names from here or is it from somewhere else?

Reply:
You get the arraylist of Category objects from the MonthMenuV, in the actionPerformed method, calling controller.getOutput(session, monthID).getCategoryData(). 

Actually, let me set session and monthID as instance variables in the constructor, so that they can be used from the actionPerformed method.

Reply:
ok I see thanks